openlibrary/core/ia.py

```diff
@@ -40,19 +42,22 @@ def get_api_response(url):
     return api_response
 
 
-def _get_metadata(itemid):
-    """Returns metadata by querying the archive.org metadata API.
+def get_metadata_direct(itemid, only_metadata=True, cache=True):
```

Comment:
Note: This previously failed when `itemid` was `None`

Reply:
It's not entirely certain this was erroring; down stream it looked like it might have been returning some sort of odd dict.